.stylelintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "stylelint-config-recommended",
```

Comment:
"Recommended" with these rules seems like a good place to start to me. Especially after glancing at [standard](https://github.com/stylelint/stylelint-config-standard) ("The standard config builds on top of the recommended config by additionally turning on over 60 of stylelint's stylistic rules with sensible defaults.").

I defer to @jimporter on whether we want to push for standard but spend time changing some defaults or simply start with recommended.

Reply:
If we want to use **stylint-config-standard**, we can fairly easily switch to the rules from https://github.com/mozilla-lockbox/lockbox-extension/issues/352#issue-276166472.
... Or any rules really. As long as we're doing some basic linting for invalid properties/values, I'm happy.